src/rules/javascript/eval_string_arg.js

```diff
@@ -0,0 +1,31 @@
+import { EVAL_STRING_ARG } from 'messages/javascript';
+import { getVariable } from 'utils';
+
+export default function(context) {
+  return {
+    CallExpression: function(node) {
+      var acceptedArgs = ['FunctionExpression', 'ArrowFunctionExpression'];
```

Comment:
We said before we were ok with long-hand for the if statement e.g. like you still have for setTimeout/setInterval. Despite being a bit more concise `indexOf` is less instantly readable. Even more so if you've got to find the Array var up here.